playbooks/common/openshift-cluster/config.yml

```diff
@@ -42,6 +42,13 @@
     r_openshift_excluder_action: disable
     r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
+- name: Disable excluders
+  hosts: oo_masters_to_config:oo_nodes_to_config
+  tasks:
+  - name: Set hostname
+    hostname: "{{ openshift.common.hostname }}"
```

Comment:
`hostname:` needs the `name:` argument
http://docs.ansible.com/ansible/latest/hostname_module.html

:+1: for moving away from `command`

Reply:
As long as this works on atomic host. I'm pretty sure we found that it didn't. 95223c4b92

Reply:
@sdodson, did we file any bugs or issues we can track for the hostname module not working?

Reply:
I don't recall.